Documentation/contributing/development/dev_setup.rst

```diff
@@ -151,8 +151,8 @@ brought up by vagrant:
   test/bpf/verifier-test.sh``.
 * ``IPV4=1``: Run Cilium with IPv4 enabled.
 * ``RUNTIME=x``: Sets up the container runtime to be used inside a kubernetes
-  cluster. Valid options are: ``docker``, ``containerd`` and ``crio``. If not
-  set, it defaults to ``docker``.
+  cluster. Valid options are: ``containerd`` and ``crio`` (``cri-o``). If not
```

Comment:
We probably don't need to state both `crio` and `cri-o` here, I see that the two work in the script, so let's just keep one of them (whichever your prefer)?

Reply:
Thanks for the quick review!

I got it, let's keep `crio`! I will fix it.